### PR TITLE
Explained the OpenShift plan offerings a bit more ref #17

### PR DESCRIPTION
**Pull request by MaiHameed**

### CODE REVIEW

**Comment by MaiHameed** [comment]
@yeekangc I rewrote the section explaining the offered plans. If this looks good to you we can push it to the master site.

**Comment by yeekangc** [comment]
Is this for a particular issue?  Can we reference that in the PR?

**Comment by yeekangc** [comment]
@Charlotte-Holt, can you please take a look at the proposed changes?

**Comment by MaiHameed** [comment]
Reference to this issue: https://github.com/openliberty/guide-cloud-openshift/issues/17

File: README.adoc [hunk 1]
```diff
@@ -72,9 +72,11 @@ Before you begin, the following additional tools need to be installed:
...
 instructions, refer to the official https://docs.docker.com/install/[Docker documentation^].
 
-- *OpenShift account:* To access a {kube} cluster, you must sign up for a Red Hat OpenShift online account. To sign up, refer to the 
-https://manage.openshift.com/register/plan[official website^]. Keep in mind that the creation time depends on resource availability 
-and may take some time.
+- *OpenShift account:* To access a {kube} cluster, you must sign up for a Red Hat OpenShift Online account. There are two options, 
+Starter and Pro. You will use the Starter plan, which includes a free trial of their platform with limited resources, perfect for individual 
+experimentation. The Pro plan includes more resources and has a monthly fee. To sign up, refer to the 
+https://manage.openshift.com/register/plan[official website^]. Keep in mind that the creation time depends on resource 
+availability and may take some time.
```
Comment (Charlotte-Holt) [inline README.adoc]: Update
"To sign up, refer to the https://manage.openshift.com/register/plan[official website^]. Keep in mind that the creation time depends on resource availability and may take some time."
To
"To sign up, go to the https://manage.openshift.com/register/plan[official website^]. Keep in mind that the creation time depends on resource availability and might take some time.

File: README.adoc [hunk 2]
```diff
@@ -72,9 +72,11 @@ Before you begin, the following additional tools need to be installed:
 supports various container types, but you will use Docker in this guide. For installation 
 instructions, refer to the official https://docs.docker.com/install/[Docker documentation^].
 
-- *OpenShift account:* To access a {kube} cluster, you must sign up for a Red Hat OpenShift online account. To sign up, refer to the 
-https://manage.openshift.com/register/plan[official website^]. Keep in mind that the creation time depends on resource availability 
-and may take some time.
+- *OpenShift account:* To access a {kube} cluster, you must sign up for a Red Hat OpenShift Online account. There are two options, 
+Starter and Pro. You will use the Starter plan, which includes a free trial of their platform with limited resources, perfect for individual 
+experimentation. The Pro plan includes more resources and has a monthly fee. To sign up, refer to the
```
Comment (Charlotte-Holt) [inline README.adoc]: Update:
"You will use the Starter plan, which includes a free trial of their platform with limited resources, perfect for individual 
experimentation."
To
"Use the Starter plan, which includes a free trial of the OpenShift platform with limited resources, making it perfect for individual experimentation."

**Comment by MaiHameed** [comment]
Updated, thanks @Charlotte-Holt

**Pull request merged**